### PR TITLE
TASK: Use Elm for Checkbox

### PR DESCRIPTION
**Pull request by mstruebing**
:trollface: 

Don't get it serious - I just wanted to try if and how that works :)

//cc @dfeyer

### CODE REVIEW

**Comment by dimaip** [comment]
Oups, it came sooner then I hoped :rofl:

File: Build/Dependencies/react-elm-components/example/Chat.elm [hunk 1]
```diff
@@ -0,0 +1,139 @@
+port module Chat exposing (main)
```
Comment (dimaip) [inline Build/Dependencies/react-elm-components/example/Chat.elm]: Why commit these examples?

Comment (mstruebing) [reply Build/Dependencies/react-elm-components/example/Chat.elm]: Like I said, it's more a joke than serious and I simply cloned the `react-elm-components`-package to change one line in their code.

**Comment by kitsunet** [comment]
No!?!

**Pull request closed without merging**